tests/baselines/reference/user/prettier.log

```diff
@@ -10,12 +10,8 @@ src/cli/util.js(454,16): error TS2339: Property 'type' does not exist on type 'n
 src/cli/util.js(455,16): error TS2339: Property 'oppositeDescription' does not exist on type 'never'.
 src/cli/util.js(456,17): error TS2339: Property 'name' does not exist on type 'never'.
 src/common/parser-create-error.js(8,9): error TS2339: Property 'loc' does not exist on type 'SyntaxError'.
-src/config/resolve-config-editorconfig.js(20,36): error TS2349: Cannot invoke an expression whose type lacks a call signature. Type 'typeof import("../../../node_modules/mem/index")' has no compatible call signatures.
-src/config/resolve-config-editorconfig.js(27,35): error TS2349: Cannot invoke an expression whose type lacks a call signature. Type 'typeof import("../../../node_modules/mem/index")' has no compatible call signatures.
-src/config/resolve-config-editorconfig.js(42,7): error TS2339: Property 'clear' does not exist on type 'typeof import("../../../node_modules/mem/index")'.
-src/config/resolve-config-editorconfig.js(43,7): error TS2339: Property 'clear' does not exist on type 'typeof import("../../../node_modules/mem/index")'.
-src/config/resolve-config.js(10,29): error TS2349: Cannot invoke an expression whose type lacks a call signature. Type 'typeof import("../../../node_modules/mem/index")' has no compatible call signatures.
-src/config/resolve-config.js(70,7): error TS2339: Property 'clear' does not exist on type 'typeof import("../../../node_modules/mem/index")'.
+src/config/resolve-config.js(12,11): error TS2571: Object is of type 'unknown'.
```

Comment:
this is new though. Probably correct? I'll have to check.

Reply:
This looks like an expected result of @weswigham's change. But I can't repro the error locally.

Reply:
I have yet to merge that change - afaik these are just from sindresorhus fixing `mem` to be importable in cjs.

Reply:
Hm. Probably, but with mem@4 I still get *different* errors than this:

```
prettier/src/config/resolve-config.js(14,12): error TS2571: Object is of type 'unknown'.
prettier/src/config/resolve-config.js(50,16): error TS2571: Object is of type 'unknown'.
prettier/src/config/resolve-config.js(51,18): error TS2571: Object is of type 'unknown'.
```

resolve-config.js hasn't changed since last July. I wonder if the `git pull` part of the user tests isn't working.